src/mcp_analyzer/fastmcp_oauth_client.py

```
@@ -0,0 +1,124 @@
+            await self._client.__aexit__(exc_type, exc_val, exc_tb)
+            self._client = None
+
+    async def get_server_info(self) -> Dict[str, Any]:
+        """
+        Fetch server information.
+
+        Returns:
+            Server information dictionary
+        """
+        if not self._client:
+            raise RuntimeError("Client not initialized. Use async context manager.")
```

Ruff (0.13.2)

Avoid specifying long messages outside the exception class

(TRY003)